pkg/operations/cordondrainvm.go

```diff
@@ -45,6 +45,7 @@ func SafelyDrainNode(az armhelpers.AKSEngineClient, logger *log.Entry, apiserver
 
 // SafelyDrainNodeWithClient safely drains a node so that it can be deleted from the cluster
 func SafelyDrainNodeWithClient(client armhelpers.KubernetesClient, logger *log.Entry, nodeName string, timeout time.Duration) error {
+	nodeName = strings.ToLower(nodeName)
```

Comment:
I asume this addresses other invocations of `SafelyDrainNodeWithClient` that weren't converting to lowercase in before calling?

Reply:
Yes. There was only one instance where we weren't coercing it to lowercase, but this should be slightly safer than fixing it before calling.

I looked at a comprehensive fix for VMSS name lowercasing as @CecileRobertMichon suggested, but there are several functions in different structs that should be treated this way and it wasn't a clear win. But I'm happy to revisit if we think a more general solution is warranted.